### ISSUE TITLE
Java 17 compatibility

### ISSUE THREAD

**Issue by LordOfTheRams** [eclipse-dali/webtools.dali#3]
Hi there!

I saw that there are some changes in the required Java version numbers. I need the DALI Tools for an old Project. Does it run on Java 17 now?

Regards

**Comment by erik-brangs** [comment]
Yes, Dali runs on Java 17 now. That's because it's part of the Eclipse IDE simultaneous release and therefore used in conjunction with other components of Eclipse. Many of those components require Java 17. Additionally, the CI infrastructure only provides certain Java releases so it's not necessarily possible to test against older Java versions. All of this applies only to the Java version used to run Eclipse itself.

It should still be possible to use Dali for applications using older Java versions.

**Comment by LordOfTheRams** [comment]
Thanks! I'll give it a try.
